src-python/lab-service/traffexam/kilda/traffexam/action.py

```diff
@@ -66,6 +67,31 @@ def _encode(self, entry):
         return Ether(src=entry.mac_address, dst=self.lldp_bcast_mac) / payload
 
 
+class LACPPush(Abstract):
+    def __call__(self, iface, push_entry):
+        pkt = self._encode(push_entry)
+
+        with self._network_namespace_context:
+            sendp(pkt, iface=iface.name, verbose=False, promisc=False)
+
+    def _encode(self, entry):
+        # entry, containing 8 boolean fields -> string representation of binary number '10011010'
+        actor_state_binary = functools.reduce(lambda a, b: a + b, map(self._bool_to_str, [entry.expired,
```

Comment:
That is not a leetcode challenge. Please write as simply as possible. use KISS principle